projects/gateway2/translator/plugins/plugins.go

```diff
@@ -111,6 +111,10 @@ type StatusPlugin interface {
 		ctx context.Context,
 		statusCtx *StatusContext,
 	) error
+	MergeStatusPlugin(
+		ctx context.Context,
+		sources any,
```

Comment:
super nit: can we have this be some interface that has the `XYZStatusPlugin` methods?

Reply:
I'm not sure I understand the ask. It's on the StatusPlugin interface.

Reply:
just to avoid any, only allow passing in things that implement status methods

nbd, just stylistic